.golangci.yaml

```diff
@@ -19,6 +19,7 @@ linters:
     - structcheck # deprecated (since v1.49.0) and replaced by 'unused'
     - sqlclosecheck # not needed for this project
     - varcheck # deprecated (since v1.49.0) and replaced by 'unused'
+    - varnamelen # too many false positives
```

Comment:
I suggest not ignoring this lint, a meaningful name is always helpful for readability purposes and should be consider as best practice, especially for multi-member teams with different experiences. And one bad habit of this go language I found is the short name which is widely used, and it influences newcomers to follow this antipattern to write code. We should prevent it to introduce into our codebase.